RobotTesting/Assets/Materials/RobotandMarkerReplacement.shader

```diff
@@ -0,0 +1,142 @@
+Shader "Custom/RobotandMarkerReplacement"
```

Comment:
When you make adjustments, rename this shader to what is does i.e. RenderWhite.shader or something to that effect. The current name indicates how it's being used, but the shader operation is much more generic.